src/plugins/recordTypes/repatriationclaim/fields.js

```diff
@@ -672,17 +678,17 @@ export default (configContext) => {
             },
           },
         },
-        nagpraDocumentationGroupList: {
+        documentationGroupList: {
           [config]: {
             view: {
               type: CompoundInput,
             },
           },
-          nagpraDocumentationGroup: {
+          documentationGroup: {
             [config]: {
               messages: defineMessages({
-                name: {
-                  id: 'field.nagpraclaims_common.nagpraDocumentationGroup.name',
+                fullName: {
```

Comment:
I think this should just be `name`.

Reply:
@ray-lee I was just testing everything and realized I switched these to `fullName` to prevent the group label from displaying because it's the same as the panel. 

Maybe it shouldn't have a defined message at all in this case?

Reply:
Ah, that makes sense. In that case just having `fullName` is fine, so that it won't appear on the record editor, but will appear when selecting the group for search on the search page. This is how Textual Inscription and Non-textual Inscription work on collectionobjects.